README.md

```diff
@@ -103,108 +103,117 @@ To learn more about Istio, please refer to the
 
 ### Architecture
 
-The pods and services that make up the Istio control plane is the first part of
-the architecture that gets installed into Kubernetes Engine, at the time we
-install the BookInfo application we simultaneously install an istio service
-proxy alongside each micro service in the application. At this point we have
+The pods and services that make up the Istio control plane are the first components of
+the architecture that get installed into Kubernetes Engine. When we
```

Comment:
that "will be" installed "on" Kubernetes engine. "An Istio service proxy is installed along with each microservice during the installation of the BookInfo application. At this point, there are two tiers that make up the Istio architecture, Control Plane and Data Plane, and microservices to manage.

Reply:
I took your suggestion and modified it more for (I think) improved readability. Tell me what you think?